### DISCUSSION TITLE
Should firing mode ignore barrel status?

### DISCUSSION THREAD

**Discussion by Toy203** [gpstar81/GPStar-proton-pack#329] [Ideas]
I had few issue with the barrel switch myself before, it's a bit temperamental. Is there a reason to closely tie the firing status to the barrel extension? I feel that by ignoring it for the firing effect the bigger issue would be not hearing the "thump" once the barrel extends.

**Comment by gpstar81** [comment]
You can just disconnect it from the board if you do not want to use it.

**Comment by Toy203** [comment]
Yes i'm aware and mine is correctly working now. It was just a thought about "regular use". Would cut off a good number of possible issues during field use i guess.
